src/contexts/SiteLaunchContext.tsx

```diff
@@ -10,10 +10,11 @@ import { launchSite } from "services/SiteLaunchService"
 
 import {
```

Comment:
> I am also mutating the database directly cuz actual site launch is slow. Will be doing a full run tmr on staging for sanity checks.

What does this mean and where is this logic?

Reply:
@harishv7 
concretely, these are the mapped DB values to what the `siteLaunchDto` will have: 

`site_status` = `INITALIZED` and `job_status` = `READY` -> `siteLaunchStatus = NOT_LAUNCHED`

`site_status` = `LAUNCHED` and `job_status` = `RUNNING` -> `siteLaunchStatus = LAUNCHING`

`site_status` = `LAUNCHED` and `job_status` = `FAILED` -> `siteLaunchStatus = FAILED`

`site_status` = `LAUNCHED` and `job_status` = `READY` -> `siteLaunchStatus = LAUNCHED`

So to get the corresponding UI in the FE, I directly mutate the values for the `siteStatus` + `jobStatus` to get the desired screens for the UI demo in this PR.

Reply:
> Ah so you are just gathering the final state of UI from a combination of DB values right? So you are not mutating the DB, you are just using DB values to formulate the end state on FE

hmm I am not sure if I understanding this statement correctly :( 
to 'mock' the entire site launch flow in the lambdas,step functions etc, rather than waiting for BE to modify the `site` row in my local db, I mutate it directly for the purposes of this demo.
so in a sense I am mutating the DB, to formulate the end state on FE

Reply:
so after the demo we are going to revert this?

Reply:
clarified - "it" refer to db values